### PR TITLE
timer

### LINKED ISSUE #6: Feruz: Timer
- [x] Timer function
- [x] Stop function
- [x] Toggle function (START <---> STOP)

### PR DESCRIPTION
**Pull request by Feruzteame**
closes #6 
hey guys, here is the timer !

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
 <head>
   <meta charset='utf8'>
   <title></title>
-
-  <link href="./public/style.css" rel="stylesheet" />
+<link href="./public/style.css" rel="stylesheet" />
+<script src="src/timer.js"></script>
```
Comment (oguzkarademir) [inline index.html:8]: @Feruzteame we are using export and import, you shouldnt write this script tag

File: index.html [hunk 2]
```diff
@@ -100,7 +100,7 @@ <h2>Created by :</h2>
 
   <!-- load your app -->
   <script type='module' src='./src/init.js'></script>
-
+<script src="src/listener.js"></script>
```
Comment (oguzkarademir) [inline index.html:103]: I guess we need to write the listeners in init.js too, there is no need to listener.js

File: src/timer.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+let min = 25
```
Comment (oguzkarademir) [inline src/timer.js:1]: We need a class and we need to write properties and methods inside, then we can create new object from this class in handler functions and use these methods.

I checked this codes and it works but timer begin from 25 : 60, how can you fix this

These are my thoughts, thanks for this big work 👏 @Feruzteame

**Review by oguzkarademir — CHANGES_REQUESTED** [review]

File: src/timer.js (hunk 3 above)
Comment (Feruzteame) [reply src/timer.js:1]: Thank you for your comment actually it is hard to work with class becouse in last module i was't attend my class, that's why i try to do it another way. But i try to fix the second.

Comment (Feruzteame) [reply src/timer.js:1]: May be if you have better idea, welcome!

**Comment by Feruzteame** [comment]
hi @oguzkarademir  and @miroslavveljanoski  i fix the time(second), now it start with 25:00.

**Pull request merged**